### ISSUE TITLE
IDEA: Make it usable through the run tool on windows google chrome computer

### ISSUE THREAD

**Issue by Jetnut** [Minesraft2/Blooket-Cheats#573]
### Gamemode

GUI.js

### Function

Allow the cheats to be ran through the run tool.

**Comment by Jetnut** [comment]
If this is possible please tell how

**Comment by ThaatDoorsGuy** [comment]
> If this is possible please tell how

i dont think that a script for a website can be run outside of the page, not to mention the browser

**Comment by Jetnut** [comment]
> > If this is possible please tell how
> 
> i dont think that a script for a website can be run outside of the page, not to mention the browser

So one of my friends said it is possible but he wont tell me how

**Comment by Minesraft2** [comment]
I don't even know what the run tool is

**Comment by Jetnut** [comment]
> I don't even know what the run tool is

push the windows button an type run

**Comment by Jetnut** [comment]
![image](https://user-images.githubusercontent.com/126018394/223781205-e2613415-5da1-4097-942b-0bda9eacf760.jpg)

**Comment by Minesraft2** [comment]
That's for opening directories and files.

**Comment by Minesraft2** [comment]
If there is actually a way, your best bet to find out is google
